clang/include/clang/Frontend/FrontendActions.h

```diff
@@ -45,6 +45,14 @@ class ReadPCHAndPreprocessAction : public FrontendAction {
   bool usesPreprocessorOnly() const override { return false; }
 };
 
+class ReadPCHAndPreprocessByModNameAction : public ReadPCHAndPreprocessAction {
```

Comment:
Does this need to inherit from `ReadPCHAndPreprocessAction`? I think `PreprocessorFrontendAction` should be just fine, since precompiled headers don't affect dependencies of a module.

Reply:
Would it be incorrect to inherit from `PreprocessOnlyAction` instead?